backend/src/services/sync/imageCredit.test.ts

```
@@ -0,0 +1,175 @@
+ */
+
+import { describe, it, expect, vi, beforeEach, afterEach } from 'vitest';
+import {
+  commonsFileName,
+  commonsFilePage,
+  creditText,
+  fetchCommonsCredits,
+} from './imageCredit.js';
+import { WaitBudget } from './sourceRetry.js';
+
+const FILE_PATH = 'http://commons.wikimedia.org/wiki/Special:FilePath/Church%20Of%20Our%20Lady%20Bruges.jpg';
```

ESLint

[error] Using http protocol is insecure. Use https instead.

(sonarjs/no-clear-text-protocols)